File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.py
class Solution:
    def getXORSum(self, arr1: List[int], arr2: List[int]) -> int:   
        ans1,ans2=0,0
        for i in arr1: ans1^=i
        for i in arr2: ans2^=i
        return ans1 & ans2
        

File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.cpp
class Solution {
public:
    int getXORSum(vector<int>& arr1, vector<int>& arr2) {
        int ans1 = 0; 
        int ans2 = 0; 
        for (auto const i:arr1) {
            ans1 ^= i;
        }
        for (auto const i:arr2) {
            ans2 ^= i;
        }
        
        return ans1&ans2;
    }
};